tests/data/h5/createNeuron.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2013-2015, EPFL/Blue Brain Project
+#                          Juan Hernando <jheranando@fi.upm.es>
+#
+# This file is part of Brion <https://github.com/BlueBrain/Brion>
+
+#!/usr/bin/env python
+
+import h5py
+import numpy
+
+SOMA = 1
+AXON = 2
+BASAL = 3
+APICAL = 4
+
+out = h5py.File('test_neuron.h5', 'w')
+
+def createTBranchPoints():
+    # Creates a branch with a T shape
+    # The first section is a non-homogeneous sampling of 10 points on a straight
```

Comment:
distances between lines are non-homogeneous, the points themselves or the sampling behaviour are homogeneous so it may be differently said ? Non-homogenus distances based on sampling. I am not totally sure you may be also right but it kind of itched my ear :)